jest.setup.js

```diff
@@ -0,0 +1,5 @@
+const setup = require('jest-setup-glob')
+
+setup({
+  '**.browser.test.ts': './test/jest.browser.setup.ts',
```

Comment:
Whenever a test file name matches this glob, the associated Jest setup will be used. Otherwise, the default setup is used.